.github/workflows/deploy.yaml

```
@@ -462,6 +462,24 @@ jobs:
           REGISTRATION_SERVICE_VERSION: 0.0.1-SNAPSHOT
           DATASPACE_DID_HOST: ${{ needs.Deploy-Dataspace.outputs.dataspace_did_host }}
 
+      - name: 'Wait for participant registration'
+        run: |
+          
+          hostDid="did:web:$DATASPACE_DID_HOST"
+          participantDid="did:web:$PARTICIPANT_DID_HOST"
+          
+          cmd="java -jar registration-service-cli-$REGISTRATION_SERVICE_VERSION-all.jar \
+                -d=$hostDid \
+                -k=participantkey.pem \
+                -c=$participantDid \
+                participants get"
+          
+          $GITHUB_WORKSPACE/system-tests/resources/cli-tools/validate_onboarding.sh "$participantDid" "$cmd"
```

```suggestion
          system-tests/resources/cli-tools/validate_onboarding.sh "$participantDid" "$cmd"
```

and 

```
        working-directory: .
```

(would be a nit, but since this pattern is used at line 409, it seems good to standardise)

---

The current addition follows the pattern of the steps in lines 419, 428, 437, 446, 451, 456, 471 which is used to call identity-hub-cli and registration-service-cli because the default working directory for the CLI is set in line 144.

```yaml
    defaults:
      run:
        working-directory: deployment/terraform/dataspace
```

However, the pattern `working-directory: .` is only used twice.

@algattik, therefore, I would not change. Makes sense?